docker-compose-clickhouse.yml

```diff
@@ -0,0 +1,52 @@
+---
+version: '3.7'
+
+services:
+  pmm-server-external-clickhouse:
+    image: ${PMM_SERVER_IMAGE:-perconalab/pmm-server:dev-latest}
+    container_name: pmm-server-clickhouse
+    depends_on:
+    - external-clickhouse
+    ports:
+      - "8080:80"
+      - "8443:443"
+    environment:
+      - PMM_DEBUG=1
+      - PERCONA_TEST_SAAS_HOST=check-dev.percona.com:443
+      - PERCONA_TEST_CHECKS_PUBLIC_KEY=RWTg+ZmCCjt7O8eWeAmTLAqW+1ozUbpRSKSwNTmO+exlS5KEIPYWuYdX
+      - PERCONA_TEST_VERSION_SERVICE_URL=https://check-dev.percona.com/versions/v1
+      - PERCONA_TEST_TELEMETRY_INTERVAL=10s
+      - PERCONA_TEST_TELEMETRY_RETRY_BACKOFF=10s
+      - PERCONA_TEST_PMM_CLICKHOUSE_ADDR=external-clickhouse:9000
+      - PERCONA_TEST_PMM_CLICKHOUSE_DATABASE=pmm
+      - PERCONA_TEST_PMM_CLICKHOUSE_POOL_SIZE=1 
+      - PERCONA_TEST_PMM_CLICKHOUSE_BLOCK_SIZE=65000
+  pmm-client-external-clickhouse:
+    image: ubuntu:latest
+    container_name: pmm-client
+    restart: on-failure
+    ports:
+      - "7778:7777"
```

Comment:
this expose is not really needed, you basically don't need to expose any ports on host for any containers, since all of them will be part of one compose, they have their own network to communicate with each other, only pmm-server ports need exposing on host